### DISCUSSION TITLE
Additional info in Navigation?

### DISCUSSION THREAD

**Discussion by Slegnor** [iaincollins/icarus#5] [Ideas & Suggestions]
Any plans to add additional info to the navigation screen?
Discovered/Mapped/1st Footfall indicators, for example?

I also feel the landable vs non-landable distinction could be clearer with the graphics, I appreciate it tries to replicate the ED UI by having the halo around the planets but its easier to spot at a glance in ED, might just be down to the predominantly blue hue of the default options?

**Comment by iaincollins** [comment]
> Any plans to add additional info to the navigation screen?
> Discovered/Mapped/1st Footfall indicators, for example?

Very much so! At least the things we can add.

> I also feel the landable vs non-landable distinction could be clearer with the graphics, I appreciate it tries to replicate the ED UI by having the halo around the planets but its easier to spot at a glance in ED, might just be down to the predominantly blue hue of the default options?

Yeah, there seems to be consensus around this, a couple of other folks didn't realize that **light blue == landable** and **dark blue == not landable**. Using a similar indicator to the in game system menu for systems with atmosphere that are landable shouldn't be a problem.
 
I've been reluctant to add the landed ship icon to planets that can be landed on as I didn't want to make it harder to pick out which ones have settlements or ports, but I think that might be the way to go. Perhaps I can use a highlight color for the settlement/port icons, and just display the lander in white.

I generally prefer to echo the in game UI, but there were situations like how it highlights ports/settlements that don't work so great so wanted to see if I could improve on that.

**Comment by iaincollins** [comment]
> I also feel the landable vs non-landable distinction could be clearer with the graphics, I appreciate it tries to replicate the ED UI by having the halo around the planets but its easier to spot at a glance in ED, might just be down to the predominantly blue hue of the default options?

Quick update to add that in v0.10.0, released yesterday, there is now an explicit icon displayed on landable planets to it much more obvious which planets can be landed on.

**Comment by alyyasser** [comment]
another detail that i would love to have is, the total bodies in the system, so i can compare what shown in icarus with my honking result. this is kinda important to me as an explorer to aid me on decision making, whether to continue further FSS the system or not.
